POC/ColorPicker/ColorPicker/TransparentWindow.xaml.cs

```diff
@@ -35,28 +68,35 @@ private void OnKeyDown(object sender, System.Windows.Input.KeyEventArgs e)
             }
         }
 
-        private void SetWindowSizeToCoverAllScreens()
+        private void OnMouseMove(object sender, System.Windows.Input.MouseEventArgs e)
         {
-            int minX = 0;
-            int maxX = 0;
+            RealignMagnificationBox(e);
+        }
 
-            int minY = 0;
-            int maxY = 0;
+        private void ConfigureUpdateTimer()
+        {
+            _updateTimer.Tick += RefreshMagnificationBox;
+            _updateTimer.Interval = new TimeSpan(1000);
+        }
 
-            foreach (Screen s in Screen.AllScreens)
-            {
-                minX = Math.Min(minX, s.Bounds.X);
-                maxX = Math.Max(maxX, s.Bounds.X + s.Bounds.Width);
+        private void RefreshMagnificationBox(object sender, EventArgs e)
+        {
+            System.Drawing.Point cursorPosition = PixelColorFinder.SafeGetCursorPos();
 
-                minY = Math.Min(minY, s.Bounds.Y);
-                maxY = Math.Max(maxY, s.Bounds.Y + s.Bounds.Height);
-            }
+            System.Drawing.Point captureCoordsTopLeft = new System.Drawing.Point(cursorPosition.X - MAGNIFICATION_WIDTH / 2,
+                                                                                 cursorPosition.Y - MAGNIFICATION_HEIGHT / 2);
 
-            Width = maxX - minX;
-            Left = minX;
+            MagnifiedImage.Source = ScreenMagnification.GetMagnificationImage(captureCoordsTopLeft,
+                                                                              MAGNIFICATION_WIDTH,
+                                                                              MAGNIFICATION_HEIGHT);
+        }
+
+        private void RealignMagnificationBox(System.Windows.Input.MouseEventArgs e)
+        {
+            System.Windows.Point position = e.GetPosition(this);
 
-            Height = maxY - minY;
-            Top = minY;
+            Canvas.SetLeft(MouseFollower, position.X + MAGNIFICATION_WIDTH / 2 + 10);
```

Comment:
What does 10 represent here?

Reply:
A lag offset to avoid the window preview appearing in itself. I will add a constant to make this intent clearer. Good catch!